src/main/kotlin/com/cognifide/gradle/aem/pkg/PackagePlugin.kt

```diff
@@ -31,11 +31,14 @@ class PackagePlugin : Plugin<Project> {
         tasks.create(UninstallTask.NAME, UninstallTask::class.java)
         val activate = tasks.create(ActivateTask.NAME, ActivateTask::class.java)
         val deploy = tasks.create(DeployTask.NAME, DeployTask::class.java)
+        val download = tasks.create(DownloadTask.NAME, DownloadTask::class.java)
+        val extractPackage = tasks.create(ExtractPackageTask.NAME, ExtractPackageTask::class.java)
```

Comment:
please follow convention. all task names consists of single world.

Reply:
Removed the 'extract' one